Ergodox EZ speedup (#5498)
qmk/qmk_firmware: quantum/debounce/readme.md

sym_g.c
sym_pk.c
sym_pr.c
sym_pr_cycles.c 
eager_g.c
eager_pk.c

> eager_pr.c //could be used in ergo-dox!

Reviewer: ```suggestion
eager_pr.c //is used in ergo-dox!
```
